client/src/components/pages/application/Signup/index.js

```diff
@@ -1,5 +1,154 @@
 import React from 'react';
+import axios from 'axios';
+import Card from '../../../common/card';
+import Typography from '../../../common/Typography';
+import Textfield from '../../../common/TextField';
+import Button from '../../../common/Button';
+import signupValidate from '../../../../utils/application/SignupValidation';
+import SelectBox from '../../../application/SelectBox';
+import './index.css';
 
-const SignUp = () => <h1>SignUp</h1>;
+import signupImage from '../../../../assets/images/signup.svg';
+
+class SignUp extends React.Component {
+  state = {
+    fullName: '',
+    email: '',
+    password: '',
+    passwordConfirmation: '',
+    location: '',
+    message: '',
+  };
+
+  handleSignin = () => {
+    window.location.href = '/signin';
+  };
+
+  handleEror = () => {
+    window.location.href = '/404';
+  };
```

Comment:
My comment above, about using withRouter to navigate, could apply here as well.

Reply:
This is what imiss..
Thanks iwill use it..